mscore/palette.cpp

```diff
@@ -685,8 +686,12 @@ void Palette::applyPaletteElement(PaletteCell* cell)
 
 void PaletteScrollArea::keyPressEvent(QKeyEvent* event)
       {
+      bool newScoreWizard = false;
       QWidget* w = this->widget();
       Palette* p = static_cast<Palette*>(w);
+      if (p->name() == "Key Signatures" && p->selectable())
```

Comment:
I think it would be better to add a boolean in the PaletteScoreArea or Palette with a good description of the behavior. Then set this boolean when creating the palette in the new score wizard.